app/src/main/java/com/door43/translationstudio/ui/spannables/USFMParagraphSpan.java

```diff
@@ -0,0 +1,27 @@
+package com.door43.translationstudio.ui.spannables;
+
+import android.text.SpannableStringBuilder;
+
+public class USFMParagraphSpan extends ParagraphSpan {
+
+    public static final String PATTERN = "\\\\p[^a-z]";
```

Comment:
Like in Desktop, I think you want a `\\p\W` here -- paragraph marker followed by a non-word character.

Reply:
Thanks! Fixed.